src/js/index.js

```diff
@@ -230,17 +238,20 @@ class ReactJsonView extends React.PureComponent {
 
         const on_edit_payload = {
             existing_src: src,
-            new_value: new_value,
-            updated_src: updated_src,
-            name: name,
-            namespace: namespace,
-            existing_value: existing_value,
+            new_value,
+            updated_src,
+            name,
+            namespace,
+            existing_value,
         };
 
         switch (type) {
         case 'variable-added':
             result = onAdd(on_edit_payload);
             break;
+        case 'variable-update-key':
```

Comment:
This is a mismatch with `this.set(rjvId, 'action', 'variable-update',{...data, type:'variable-key-added'});` in Object attributes line 70. Is this intended?